windshaft/server.js

```diff
@@ -8,45 +9,17 @@ var dbPassword = process.env.DRIVER_DB_PASSWORD;
 var redisHost = process.env.DRIVER_REDIS_HOST;
 var redisPort = process.env.DRIVER_REDIS_PORT;
 
-// RFC4122. See: http://stackoverflow.com/questions/7905929/how-to-test-valid-uuid-guid
-var uuidRegex = /^[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$/i;
-
-var baseQuery = ["(SELECT * FROM ashlar_record WHERE schema_id = ",
-                 "(SELECT uuid FROM ashlar_recordschema ",
-                 "WHERE next_version_id IS NULL"
-                ].join("");
-var filterQuery =  " AND record_type_id = '";
-var endQuery = ")) AS ashlar_record";
-
 var config = {
         useProfiler: true,
-        base_url: '/tiles/recordtype/:recordtype',
-        base_url_notable: '/tiles/recordtype/:recordtype',
+        // :recordtype parameter can be:
+        // GEO to get the user-uploaded boundary polygon;
+        // ALL to get record points of all record types; or,
+        // a UUID of a particular record type (returns record points of that type)
+        base_url: '/tiles/:recordtype',
```

Comment:
Ok. I'd considered using `type` for the parameter name, but decided not to, as it is a reserved keyword in JavaScript. I'll figure something else out that's more general.